VAMobile/src/constants/screens.ts

```diff
@@ -23,7 +22,11 @@ export const LARGE_PANEL_OPTIONS: StackNavigationOptions = {
   ...(isIOS() && store.getState().accessibility.isVoiceOverTalkBackRunning
     ? {
         ...TransitionPresets.ModalSlideFromBottomIOS,
-        cardStyle: { borderRadius: 0, paddingTop: '30%', backgroundColor: theme.colors.background.overlayOpacity },
+        cardStyle: {
+          borderRadius: 0,
+          paddingTop: '30%',
+          backgroundColor: '#0000004D', // Black with 30% opacity
```

Comment:
If it works the same, maybe preferable to use black + the [opacity](https://reactnative.dev/docs/view-style-props#opacity) prop.

Reply:
It's been a while, so I can't remember exactly, but I think the reason I had to set the opacity via the color's hex code is because when you set it manually via the `opacity` prop, weird things happen when a user has the screen reader enabled. I'll have to revisit this and see if that's still the case 🤔